tad/api/routes/tasks.py

```diff
@@ -0,0 +1,48 @@
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends, Request, status
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from tad.models.task import MoveTask
+from tad.services.tasks import TasksService
+
+router = APIRouter()
+templates = Jinja2Templates(directory="tad/site/templates")
```

Comment:
it is propably better to only load the templates in one place so we can easily edit it if needed. for example to the deps file

Reply:
i would  recommmende adding the autoescape to tje jinja2Template function: env={"autoescape": True}